runtime/lua/vim/keymap.lua

```diff
@@ -79,6 +81,9 @@ function keymap.set(mode, lhs, rhs, opts)
   end
 end
 
+---@class KeymapDelOpts
```

Comment:
Can't see where this is used. Also needs `vim.` prefix.

Reply:
Good catch. I don't think we need it anymore.